install.sh

```diff
@@ -61,6 +61,9 @@ make_virtualenv() {
     source $VENV_DIR/$1/bin/activate
     python3 -m pip install --upgrade pip setuptools wheel
 
+    if [ -f "pre_requirements.txt" ]; then
+        python3 -m pip install --upgrade -r pre_requirements.txt
```

Comment:
why the need of the new file, can't we add `setuptools<=57.0.5` to the existing `tap-adwords/requirements.txt` ?

Reply:
makes sense, thanks